main.tf

```diff
@@ -409,7 +401,32 @@ resource "helm_release" "castai_kvisor" {
   }
 
   set {
-    name  = "structuredConfig.provider"
+    name  = "castai.grpcAddr"
+    value = var.api_grpc_addr
+  }
+
+  set {
+    name  = "controller.replicas"
+    value = var.install_security_agent == true ? 1 : 0
```

Comment:
This is required if helm will be managed like evictor https://github.com/castai/terraform-castai-gke-cluster/blob/c67c3da712b927515c0bdab0881024115e27fa43/main.tf#L327